jackson-databind/src/main/java/io/micronaut/jackson/ObjectMapperFactory.java

```diff
@@ -143,6 +147,10 @@ public ObjectMapper objectMapper(@Nullable JacksonConfiguration jacksonConfigura
             objectMapper.findAndRegisterModules();
         }
         objectMapper.registerModules(jacksonModules);
+        if (beanContext != null) {
```

Comment:
can this be null?

Reply:
There are some cases where `ObjectMapperFactory` is used without the bean context